Documentation/Articles/querying_for_data.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Querying for Data
+description: About Reading data
+keywords: read model, query, ddd, read, cqrs,  
+author: pavsaund
+---
+
+
+# Querying for Data
```

Comment:
I'd prefer something like "Querying your System" rather than data.

Reply:
Resolving as I go here - not sure if this is the way to do things.